_data/bootloaders.json

```diff
@@ -29,10 +29,19 @@
             "family": "esp32s2",
             "bootloader_id": "adafruit_feather_esp32s2_tft",
         },
+        "adafruit_feather_rp2040": {
+            "family": "raspberrypi",
```

Comment:
Is there supposed to be a `raspberrypi` entry in the `bootloaders` object at the beginning of this file? It seems like the rest of the values used for `family` are also present in that object.

Reply:
There's no bootloader built for `raspberrypi` family MCUs (it's in the ROM), so leaving it out seems okay unless it's to be used as a conditional in `_includes/download/board.html` (perhaps for a note like "it's in the ROM"?) like the other bootloader values.